design/schedules.md

```diff
@@ -0,0 +1,76 @@
+# Schedules
+
+We already support `trigger` and (soon) `pr` triggers.
+This spec covers scheduled triggers.
+By way of introduction, here's a rich example of the syntax:
+
+```yaml
+schedules:
+- cron: "0 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+    - releases/*
+    exclude:
+    - releases/ancient/*
+- cron: "0 12 * * 0"
+  displayName: Weekly Sunday build
+  branches:
+    include:
+    - releases/*
+  batch: true
+  always: true
+```
+
+These definitions say:
+- For `master` and all `releases/*` branches except those under `releases/ancient/*`, run a pipeline at midnight every day.
+- For all `releases/*` branches, run a pipeline at noon on Sundays.
+  - Only run against branches which have changed (the `always` keyword).
+  - Only run one instance of the pipeline per branch (the `batch` keyword). If a branch already has an active run, queue the pipeline instead of running it immediately.
```

Comment:
do we need `batch`? aren't queues the throttling mechanism? Either demand a specific agent or create a separate queue for throttling.

Reply:
We need the behavior of "I want to only run one of these at a time; queue if there's one already running". Much like `batch: true` on `ci`, right?